components/Location.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import styled from 'styled-components';
+<<<<<<< HEAD
```

Comment:
@ericyoungson : Think these are introduced from a merge conflict. You'll need to resolve the conflicts before pushing. 🤔